index.html

```diff
@@ -292,5 +292,47 @@ <h3 class="skillset-heading">Skills</h3>
     </li>
   </ul>
 </section>
+<!-- CONTACT SECTION  -->
+<section class="contact">
+  <div class="contact-container container">
+    <div class="contact-desc">
+      <h2 class="contact-title">Contact me</h2>
+      <p class="contact-detail">
+        If you have an application you are interested in developing, a
+        feature that you need built or a project that needs coding. I’d love
+        to help with it!
+      </p>
+    </div>
+    <form
+      action="https://formspree.io/f/xknaewkq"
+      method="post"
+      class="contact-form"
+    >
+      <input
+        class="contact-input"
+        type="text"
+        maxlength="30"
+        name="user-name"
+        placeholder="Lorem Ipsum"
+        required
+      />
+      <input
+        class="contact-input"
+        type="email"
+        name="user-email"
+        placeholder="Loremipsum@gmail.com"
+        required
+      />
+      <textarea
+        class="contact-message"
+        name="user-message"
+        maxlength="500"
+        placeholder="Write the message here"
+        required
+      ></textarea>
+      <button class="contact-btn" type="submit">Get in touch</button>
```

Comment:
- [ ] Kindly consider displaying this button on the left side not in the center as you did to implement exactly the provided design check the image below for more clarification
- your design 👇 
<img width="376" alt="Screen Shot 2023-05-10 at 10 58 47" src="https://github.com/kkmanuu/My-Portfolio/assets/60146030/585c8f24-0c9c-43eb-bb32-30330bd3c123">


- Figma design 👇 
<img width="364" alt="Screen Shot 2023-05-10 at 11 00 13" src="https://github.com/kkmanuu/My-Portfolio/assets/60146030/edc368f4-b78c-4636-963a-f85ea4a076eb">

Reply:
Done in displaying the button to the left.